Unit3/Module3.1/Mini-project3.1b/README.md

```diff
@@ -0,0 +1,10 @@
+# Mini Project 3.1b: Edit Existing Products
+
+##### Credit to OpenClassrooms
+![Become](https://github.com/OCclassprojects/logo/blob/master/fav-icon.png?raw=true)
+
+### Mini Project Instructions
+
+1. Open the [project page on OpenClassrooms](https://openclassrooms.com/en/courses/4286486-build-web-apps-with-reactjs/4317921-practice-extend-the-form-functionality-to-edit-existing-products)
+
+2. Create a new branch and then new Pull Request and assign it to your coach for review.
```

Comment:
Again, same as above, lets tell them what specific files they should be putting into the PR.